test/integration/admin_login_test.rb

```diff
@@ -0,0 +1,45 @@
+require 'test_helper'
+
+class GuestUserTest < ActionDispatch::IntegrationTest
+  include Capybara::DSL
+
+  attr_reader :user, :item1, :item2, :category1, :category2
+
+  def setup
+    @user = User.create(username: 'user',
+                password: 'password',
+                first_name: 'John',
+                last_name: 'Doe',
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.